price-feeder/config/config.go

```diff
@@ -59,6 +63,28 @@ type (
 		Quote     string   `toml:"quote" validate:"required"`
 		Providers []string `toml:"providers" validate:"required,gt=0,dive,required"`
 	}
+
+	Account struct {
+		ChainID   string `toml:"chain_id" validate:"required"`
+		Address   string `toml:"address" validate:"required"`
+		Validator string `toml:"validator" validate:"required"`
+	}
+
+	Keyring struct {
+		Backend string `toml:"backend" validate:"required"`
+		Pass    string `toml:"pass" validate:"required"`
+		Dir     string `toml:"dir" validate:"required"`
+	}
+
+	RPC struct {
+		TMRPCEndpoint string `toml:"tmrpc_endpoint" validate:"required"`
+		GRPCEndpoint  string `toml:"grpc_endpoint" validate:"required"`
+		RPCTimeout    string `toml:"rpc_timeout" validate:"required"`
+	}
+
+	Gas struct {
```

Comment:
I'm not sure if Gas warrants its own struct -- we can just probably add `GasAdjustment` directly to `Config`.